src/Module/Photo.php

```diff
@@ -100,11 +100,6 @@ protected function rawContent(array $request = [])
 				$id = $account['id'];
 			}
 
-			// Contact Id Fallback, to remove after version 2021.12
```

Comment:
I guess I left this in, since we do have parts in the code that still refer to that path schema.

Reply:
It's worse than that, there's no replacement feature. I'm not even sure what was supposed to be the replacement feature.